### ISSUE TITLE
GIS visualization crashes on MacOS for Intel install

### ISSUE THREAD

**Issue by mdozier-a** [dbeaver/dbeaver#19661]
### Description

The app instantly crashes when clicking on a geometry value when in the Spatial tab.

I have tried resetting my UI both in the app and manually, doing a complete uninstall and reinstall. Still crashing. 

logs: 
```
!ENTRY org.eclipse.ui 2 2 2023-04-10 15:00:00.929
!MESSAGE Invalid preference category path: org.eclipse.debug.ui.ConsolePreferencePage (bundle: org.eclipse.ui.console, page: org.eclipse.ui.internal.console.ansi.preferences.AnsiConsolePreferencePage)
!SESSION 2023-04-10 15:09:11.432 -----------------------------------------------
eclipse.buildId=unknown
java.version=17.0.6
java.vendor=Eclipse Adoptium
BootLoader constants: OS=macosx, ARCH=x86_64, WS=cocoa, NL=en_US
Framework arguments:  -keyring /Users/***/.eclipse_keyring
Command-line arguments:  -os macosx -ws cocoa -arch x86_64 -keyring /Users/***/.eclipse_keyring

!ENTRY org.eclipse.core.resources 2 10035 2023-04-10 15:09:14.350
!MESSAGE The workspace exited with unsaved changes in the previous session; refreshing workspace to recover changes.

!ENTRY org.eclipse.ui 2 2 2023-04-10 15:09:16.514
!MESSAGE Invalid preference category path: org.eclipse.debug.ui.ConsolePreferencePage (bundle: org.eclipse.ui.console, page: org.eclipse.ui.internal.console.ansi.preferences.AnsiConsolePreferencePage)
```

### DBeaver Version

community 23.0.2

### Operating System

macOS ventura 13.1

### Database and driver

Postgresql

### Steps to reproduce

Load geo datas on a postgis database 
go to spatial view
click on an entry
app crashes

### Additional context

SSH tunnel connection

**Comment by ShadelessFox** [comment]
Hello @mdozier-a,

Please post JVM crash logs instead. Please see https://github.com/dbeaver/dbeaver/wiki/Log-files#java-fatal-logs

**Comment by Destrolaric** [comment]
@mdozier-a The issue is fixed in the macOS Ventura 13.2.
